client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts

```diff
@@ -45,10 +45,7 @@ export const useUrlQueryParams = ({
     if (sort !== currentSort) {
       updateQuery({ sort, dir: '', page: '' });
     } else {
-      const dir =
-        column.sortBy?.direction === 'asc' || !column.sortBy?.direction
-          ? 'desc'
-          : '';
+      const dir = column.sortBy?.direction === 'desc' ? '' : 'desc';
```

Comment:
same? but less code.. 
The expected values are `'desc'` `'asc'` `undefined` and we just want to toggle

Reply:
Yep, that does look better !